tensorboard/webapp/persistent_settings/persistent_settings_module.ts

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {NgModule} from '@angular/core';
+
+import {PersistentSettingsConfigModule} from './persistent_settings_config_module';
+import {PersistentSettingsDataSourceModule} from './_data_source/persistent_settings_data_source_module';
+
+/**
+ * Persistent Settings module is responsible for persisting and loading settings
+ * from other features.
+ *
+ * For settings of the app, please refer to "settings" feature instead.
```

Comment:
I cannot understand at once from this docString. what are "other features" and what are "settings of the app"
Can we add some examples on the description?

Reply:
Left a comment in the code. PTAL.